dom.bs

```diff
@@ -1419,6 +1419,103 @@ that gave folks all the wrong ideas. <a>Events</a> do not represent or cause act
 can only be used to influence an ongoing one.
 
 
+<h3 id=controlling-promises>Controlling {{Promise}}s</h3>
```

Comment:
`CancellationController`? That seems reasonable as a base class for `FetchController` if @jakearchibald needs such a thing.